dpytools/email/ses/client.py

```diff
@@ -0,0 +1,86 @@
+import re
+import boto3
+from botocore.exceptions import BotoCoreError, ClientError
+
+class SesClient:
+    """
+    A client for sending emails using AWS SES and the boto3 client.
+
+    Attributes:
+        client: The boto3 SES client.
+        sender: The sender's email address.
+    """
+
+    EMAIL_REGEX = r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
```

Comment:
have a look at this: https://github.com/JoshData/python-email-validator , regex works but is fragile and really not that obvious. If someone wants to solve this for us, lets let them.